stdlib/http/src/main/ballerina/src/http/http_errors.bal

```diff
@@ -197,12 +202,12 @@ public type InboundRequestError InitializingInboundRequestError|ReadingInboundRe
 # Defines the listener error types that returned while sending outbound response
 public type OutboundResponseError InitializingOutboundResponseError|WritingOutboundResponseHeadersError|
                             WritingOutboundResponseBodyError|Initiating100ContinueResponseError|
-                            Writing100ContinueResponseError;
+                            Writing100ContinueResponseError|CookieHandlingError;
```

Comment:
`OutboundResponseError` is occurred at listener-side, while `ClientError` is occurred at the client-side. It does not seem correct to have the same error in both `OutboundResponseError` and `ClientError` types.

As per the discussion, shall we create a new Error type for Cookie-related errors in the listener-side ?